assets/app/views/create.html

```diff
@@ -26,13 +25,13 @@
                   </span>
                   <span class="input-group-btn">
                     <button type="submit" class="btn btn-lg btn-default" ng-click="createFromSource()"
-                      ng-disabled="from_source_form.$invalid" value="">Next</button>
+                      ng-disabled="from_source_form.from_source_url.$error.required" value="">Next</button>
```

Comment:
can't we keep this as `$invalid`? that should only trigger off of `required` now, right?